docs/book/src/developer/tilt.md

```diff
@@ -507,6 +507,28 @@ syntax highlighting and auto-formatting. To enable it for Tiltfile a file associ
 
 NB: The socket defined by `DOCKER_HOST` is used only for the `hack/tools/internal/tilt-prepare` command, the image build is running the `podman build`/`podman push` commands.
 
+## Using Lima
+
+[Lima] can be used instead of Docker. Please note that especially with CAPD the rootless template of Lima does not work.
```

Comment:
```suggestion
[Lima] can be used instead of Docker Desktop. Please note that especially with CAPD the rootless template of Lima does not work.
```
nit: We still get Docker